client/js/app.modules/main/main.module.js

```diff
@@ -0,0 +1,8 @@
+var angular = require('angular');
+
+var mainModule = angular.module('mainModule', [])
+    .controller('mainCtrl', require('./main.controller.js'))
```

Comment:
I suggest to stick to UpperCamelCase convention in naming controllers, because they are constructors.
This kind of naming described in popular angular style guide:
https://github.com/johnpapa/angular-styleguide#style-y123
But its up to you guys, the main point is that whole team should use same naming convention for angular entities.